src/LoginRouter.js

```diff
@@ -170,6 +170,15 @@ function (BaseLoginRouter,
       this.render(PrimaryAuthController, { Beacon: SecurityBeacon });
     },
 
+    wildcard: function (route) {
+      if(route == 'login/login.htm') {
```

Comment:
nit: space after if ```if (``` and maybe use ```===``` instead?

Reply:
👍 Sorry, removed === by accident during my testing